pkg/evpn/svi.go

```diff
@@ -213,7 +246,7 @@ func (s *Server) GetTunnel(_ context.Context, in *pb.GetTunnelRequest) (*pb.Tunn
 		return nil, err
```

Comment:
We could have an util funtion for initial validation instead of repeating the same the every SVI create/update/del function.

Reply:
for sure, will start refactoring after this one, after switch is completed